agent_build_refactored/managed_packages/managed_packages_builders.py

```diff
@@ -81,10 +74,26 @@
 EMBEDDED_PYTHON_VERSION = "3.11.0"
 EMBEDDED_PYTHON_SHORT_VERSION = ".".join(EMBEDDED_PYTHON_VERSION.split(".")[:2])
 
-PYTHON_PACKAGE_SSL_VERSION = "1.1.1k"
+PYTHON_PACKAGE_SSL_1_1_1_VERSION = "1.1.1k"
```

Comment:
Is this the latest 1.1 version we can use?

Reply:
There's, actually, a new version 1.1.1s :) The new job that scan's for openssl updates is still waits to be added.

Reply:
Sorry, I wanted to use if 1.1.1k is the latest version we can use due to the compatibility reasons or we can use latest available version out there (s or k)?

Reply:
As I understood from my observations, all patch versions of 1.1.1 (aka 1.1.1[a-z]) have to be binary compatible. For the Python it also does not matter what patch version is used, it just has to be 1.1.1+, and it can resolve everything automatically.

Reply:
Ack, thanks. In this case it would likely make sense for us to try to use the latest available version.